Swiftfin/Components/AlphaPicker/AlphaPickerButton.swift

```diff
@@ -0,0 +1,90 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2023 Jellyfin & Jellyfin Contributors
+//
+
+// Joe Kribs: joseph@kribs.net 02/06/2023
```

Comment:
?

Reply:
Whoops! That's from some X-Code defaults I was messing around with when I first started. Hence the February (8 months ago). 

It'll be removed on the next commit.